docs/SCENARIOS.md

```
@@ -149,7 +173,27 @@ scenarios:
+
+The runner executes plain `rd` and `rundown` commands directly through the current CLI entry point instead of through a shell. Leading command-scoped environment assignments are supported:
+
+```yaml
+commands:
+  - RD_AGENT_ID=agent-a RD_SESSION_ID=session-a rd claim ${TOKEN}
+  - RD_AGENT_ID=agent-a RD_SESSION_ID=session-a rundown pass
+```
+
+Shell operators in an `rd`/`rundown` command are rejected (`&&`, `||`, `|`, etc.). Split those into separate `commands` entries. Non-`rd` commands run through the user's shell.
+
+Prefix a command with `! ` when a non-zero exit is the expected outcome:
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)